app/src/main/java/com/github/se/travelpouch/ui/dashboard/map/ActivitiesMapScreen.kt

```diff
@@ -156,9 +219,215 @@ fun ActivitiesMapScreen(
                 Icon(imageVector = Icons.Filled.ArrowBack, contentDescription = "Go Back")
               }
         }
-      })
+      }
+}
+
+// -------------------------------------------------
+// Helper Functions for Displaying Information in Bottom Sheet
+// -------------------------------------------------
+
+/**
+ * Displays detailed information about an activity, including:
+ * - Title, date, and time.
+ * - Location name and description.
+ *
+ * @param activity The activity to display.
+ */
+@Composable
+fun ActivityDetailsContent(activity: Activity) {
+  // Date formatter
+  val dateFormatter = SimpleDateFormat("dd/MM/yyyy", Locale.getDefault())
+  val formattedDate = dateFormatter.format(activity.date.toDate())
+
+  // Time formatter
+  val timeFormatter = SimpleDateFormat("HH:mm", Locale.getDefault())
+  val formattedTime = timeFormatter.format(activity.date.toDate())
+
+  // Create a nested scroll connection
+  val nestedScrollConnection = remember { object : NestedScrollConnection {} }
+
+  LazyColumn(
+      modifier =
+          Modifier.fillMaxWidth()
+              .fillMaxHeight(0.3F)
+              .padding(16.dp)
+              .nestedScroll(nestedScrollConnection) // Add nested scroll connection
```

Comment:
Nice job. The column is not too big, and works smoothly. Also, great idea to put a nestedScrollConnection